### PR TITLE
errors

### PR DESCRIPTION
**Pull request by bouboumael**

### CODE REVIEW

**Review by wonecode — DISMISSED** [review]

File: templates/bundles/TwigBundle/Exception/error500.html.twig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    {{ parent() }}
+    - Site indisponible
+{% endblock %}
+
+{% block body %}
+    <main class="p-md-5 p-4">
+        <div class="p-md-5 pd-3">
+            <h1 class="h1 text-danger">Erreur 500 - Site temporairement indisponible</h1>
+            <h2 class="h3">Le site est actuellement indisponible. Merci de réessayer ultérieurement.</h2>
+            <p>Si le problème persiste, merci de contacter l'administrateur du site.</p>
```
Comment (LucyMerlier) [inline templates/bundles/TwigBundle/Exception/error500.html.twig]: but how do you contact the admin? + they should always (ideally) contact the admin, not only if the error happens again

File: templates/bundles/TwigBundle/Exception/error500.html.twig [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+{% block title %}
+    {{ parent() }}
+    - Site indisponible
+{% endblock %}
+
+{% block body %}
+    <main class="p-md-5 p-4">
+        <div class="p-md-5 pd-3">
+            <h1 class="h1 text-danger">Erreur 500 - Site temporairement indisponible</h1>
+            <h2 class="h3">Le site est actuellement indisponible. Merci de réessayer ultérieurement.</h2>
```
Comment (LucyMerlier) [inline templates/bundles/TwigBundle/Exception/error500.html.twig]: a http 500 code does not necessarily mean the site is completely unavailable, it means there was an error on a specific page that (ideally) needs to be reported => you could ideally provide a link to a contact page, and a link to go back to home page

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: templates/bundles/TwigBundle/Exception/error500.html.twig (hunk 2 above)
Comment (bouboumael) [reply templates/bundles/TwigBundle/Exception/error500.html.twig]: ok

File: templates/bundles/TwigBundle/Exception/error500.html.twig (hunk 1 above)
Comment (bouboumael) [reply templates/bundles/TwigBundle/Exception/error500.html.twig]: ok

**Review by jaldabaoth-code — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**